src/projects/EnsureThat/Extensions/ComparableExtensions.cs

```diff
@@ -1,13 +1,14 @@
 using System;
+using JetBrains.Annotations;
 
 namespace EnsureThat.Extensions
 {
     internal static class ComparableExtensions
     {
-        internal static bool IsLt<T>(this IComparable<T> x, T y) => x.CompareTo(y) < 0;
+        internal static bool IsLt<T>([NotNull] this IComparable<T> x, T y) => x.CompareTo(y) < 0;
```

Comment:
Same here, I do get the value of using it everywhere but...not sure I like it on internal APIs.